contracts/vaults/RibbonThetaVault.sol

```diff
@@ -312,7 +313,7 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
 
         emit Deposit(msg.sender, amount, currentRound);
 
-        VaultDeposit.DepositReceipt storage depositReceipt =
+        VaultDeposit.DepositReceipt memory depositReceipt =
```

Comment:
doesn't memory create a new copy? 

https://www.geeksforgeeks.org/storage-vs-memory-in-solidity/